src/main/react-front-end/src/App.css

```diff
@@ -261,42 +258,102 @@ div.homePage {
 /*    For Queue/QueueComponent.js   */
 /*                                  */
 /************************************/
-.QueueView {
-  margin-left: 10%;
-  margin-right: 10%;
+
+.screenShade {
+  position: fixed;
```

Comment:
should this not be a position relative so that we support different kinds of browsers with consistent look?

Reply:
I believe this one is fine because it is a shade background to blur out the background, and should be consistent on any device since the height and the width is taken from the user's view port